spec/chewy/type/adapter/active_record_spec.rb

```diff
@@ -63,7 +65,10 @@
   describe '#import' do
     def import(*args)
       result = []
-      subject.import(*args) { |data| result.push data }
+      subject.import(*args) do |data|
+        result.push data
+        yield if block_given?
+      end
       result
     end
```

Comment:
This is `Array#each` behavior to return self despite block is given or not. [Enumerator#each](http://ruby-doc.org/core-2.2.0/Enumerator.html#method-i-each) behaves differently when block is given.